src/cmd-push-container

```diff
@@ -40,6 +40,9 @@ if args.authfile is None:
     args.authfile = os.environ.get("REGISTRY_AUTH_FILE")
 if args.authfile is not None:
     skopeoargs.extend(['--authfile', args.authfile])
-skopeoargs.extend([f"oci-archive:{ociarchive}", f"docker://{args.name}"])
+container_name = args.name
+if ":" not in container_name:
```

Comment:
Longer term I think it'd be a bit more elegant to use an API to parse the container reference but...certainly fine for now.

Maybe hoist things out so it's like:
```
default_tag = "latest"
if ":" not in container_name:
    default_tag = f"{latest_build}-{arch}"
```
or so?  But anyways that can be a followup.

Reply:
Enabled auto merge, feel free to let that go through and do any followup, or cancel it and re-push here if you like.

Reply:
One thing I would say is that I think `-{arch}` suffix is hopefully going to be replaced with proper manifest lists.  As of today OCP builds don't support that directly, which is one reason we use the arch-tagging hack.  But, again I think (really hope) that's going to be fixed.